lib/gitlab/client/versions.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Gitlab::Client
+  # Defines methods related to version
+  # @see https://docs.gitlab.com/ce/api/version.html
+  module Versions
```

Comment:
I tried to follow the existing pattern for API calls. Please let me know if there is a more appropriate location for this method call.